src/thor/timedep_reverse.cc

```diff
@@ -54,245 +33,20 @@ void TimeDepReverse::Init(const midgard::PointLL& origll, const midgard::PointLL
   // Reserve size for edge labels - do this here rather than in constructor so
   // to limit how much extra memory is used for persistent objects.
   // TODO - reserve based on estimate based on distance and route type.
-  edgelabels_rev_.reserve(kInitialEdgeLabelCount);
+  edgelabels_.reserve(kInitialEdgeLabelCount);
```

Comment:
This whole Init() thing can likely go away if we can use the BDEdgeLabel object for the forward search.